Header header
string nome
uint8 eta
string corso_di_laurea
